index.bs

```diff
@@ -587,6 +587,44 @@ See also:
 * [[#secure-context]]
 * [[#consent]]
 
+<h3 id=text-formats>Make textual formats for people</h3>
+
+Design textual formats to be both produced and consumed by people.
+
+People who are presented with a textual format will
+expect to be able to use a text editor
```

Comment:
```suggestion
People who are presented with a textual format
should be able to use a text editor
```

It’s not about expectation. People often _expect_ terrible user experiences — that doesn't make them okay.